azurerm/resource_arm_mariadb_virtual_network_rule_test.go

```diff
@@ -261,7 +261,7 @@ resource "azurerm_mariadb_server" "test" {
   resource_group_name          = "${azurerm_resource_group.test.name}"
   administrator_login          = "acctestun"
   administrator_login_password = "H@Sh1CoR3!"
-  version                      = "5.6"
+  version                      = "10.2"
```

Comment:
Ah, thanks @katbyte! Still working out app credentials to get acceptance tests to run on our end.